cmd/podman/start.go

```diff
@@ -62,14 +62,10 @@ func startCmd(c *cliconfig.StartValues) error {
 		return errors.Errorf("you cannot start and attach multiple containers at once")
 	}
 
-	sigProxy := c.SigProxy
+	sigProxy := c.SigProxy || attach
```

Comment:
The old version had the advantage of detecting is `--sig-proxy` was explicitly set when `--attach` was not set and making that an error

Reply:
This preserves that behavior: see below. (The `.Changed` check is no longer necessary, because we are now guaranteed that the condition in line 67 can only happen if user explicitly sets `--sig-proxy` without `--attach`).

Reply:
Why the default change from false to true for sig-proxy?

Reply:
My intention is to preserve the existing behavior, as both documented and implemented. I believe that my changes preserve everything exactly as before. The only difference is that the Cobra default is now false instead of true; the purpose of that is to change the autogenerated --help message:
```
   old: --sig-proxy    Proxy received signals to the process (default true if attaching, false otherwise) (default true)
   new: --sig-proxy    Proxy received signals to the process (default true if attaching, false otherwise)
```
That is, remove the confusing duplicate '(default true)' from the usage message. AFAICT there is no way to tell Cobra "I've got the `default` message covered, please shut up"

Reply:
Oh oh oh... I think I see what's going on here. Sorry it took a minute, I need more coffee.

Yeah, this works fine.

Reply:
If @mheon's hip, I'm hip.  Too many reviews today.